contracts/vaults/balancer/external/ComposableAuraHelper.sol

```diff
@@ -98,26 +98,21 @@ library ComposableAuraHelper {
         );
         bool isSingleSidedExit = params.redemptionTrades.length == 0;
         
-        uint256 bptToSettle = context.baseStrategy._getEmergencySettlementParams({
-            maturity: maturity, 
-            totalPoolSupply: context.oracleContext.virtualSupply
-        });
-
         /// @notice minAmounts are not required to be passed in by the caller for this strategy vault
         uint256[] memory minAmounts = context.poolContext._getMinExitAmounts({
```

Comment:
@weitianjie2000 this call to getMinExitAmounts will call checkPriceLimit for the token pair and revert if the spot price in the Balancer pool is too different from the oracle price.

I worry that this could prevent us from executing in the event of an emergency where you would think that there's a greater than normal likelihood that the spot price is different from the oracle price.

I think we should either pass in the minAmounts to the emergency exit function or just hardcode them to zero and hardcode isSingleSidedExit to False.

Reply:
done